### ISSUE TITLE
Search field don't display anything

### ISSUE THREAD

**Issue by hinderer-matthieu** [TypeRocket/typerocket#101]
Hello,

I recently updated typerocket and my search fields don't display anything when I type in them.
I have the version 3.0.54 of typerocket.

Here's my code who used to work:

`
$product = $tbo_plan_information_form->search('product');
$product->setPostType('tbo_product');
$product->setLabel( __( 'Product', 'tbo' ) );
`

Thanks for your help!

**Comment by kevindees** [comment]
Hey @hinderer-matthieu 

Have you updated the assets as well and flushed your cache? https://typerocket.com/docs/v3/updating/#section-assets

```php
$product = $tbo_plan_information_form->search('product');
$product->setPostType('tbo_product');
$product->setLabel( __( 'Product', 'tbo' ) );
echo $product; // not sure if you are outputting the field
```

Thanks,
Kevin

**Comment by GPurdue** [comment]
Hi guys, I'm having the same problem. I've updated the assets but noticed that in the chrome console when I type in the search field I get an error:

`http://localhost/wp-json/typerocket/v1/search?post_type=any&s=home 404 (Not Found)`

My site is running in the folder http://localhost/typerocket/ Does this shed any light on the situation?

Should it not be looking somewhere like this? 
`http://localhost/typerocket/wp-json/typerocket/v1/search?post_type=any&s=home`

Cheers
Gary

**Comment by kevindees** [comment]
Hey @GPurdue 

Yes, it looks like the search field is looking in the wrong place. This may be a bug if you are using WordPress within a path of the root domain. I'll look more into this and see what can be done.

Thanks,
Kevin

**Comment by kevindees** [comment]
Hey @GPurdue 

This bug has been addressed in version 4.0 beta. 4.0 is coming out soon but requires PHP 7.0.

Thanks,
Kevin
